cape-ph/cape-cod
# added private swimlane vpn

# TO TEST
**NOTE: This branch is currently deployed. If you want to test the deployment, you'll need to destroy/up AND update the pre-configured box ovpn file as a new deployment will hose the one currently in there**

* read everything.
* perform the steps in the vpn readme to get the ovpn file and update it with the client key/cert. 
  * we have a pre-configured ovpn file in box, but need to make sure the procedure in the readme is coherant.
  * The client key/cert are also in box. ask @thecaffiend for the location
* connect to vpn. 
  * you do not need to be logged into aws for this to work
  * make sure your internet connection still works and you can ping various assets. your machine's DNS will be in use, not one in the VPN setup
  * make sure you have an ip address in the range `10.1.0.0 - 10.1.3.255` for a `tun` interface
  * make sure you can ping `10.0.3.1` (the gateway for the vpn subnet
  * make sure you **cannot** ping `10.0.2.1` (the gateway for the compute subnet) nor `10.0.1.1` (the gateway for the public subnet)
* approve PR
